spec/unit/cli_spec.rb

```diff
@@ -660,6 +660,23 @@
   end
 
   describe "#error_handling" do
+    context "OptionParser" do
```

Comment:
I'd prefer this to be a bit more descriptive, if just OptionParser will be mentioned then I feel no need to have the context block. On the other hand if we say why we need to group these tests with something like "for options" (or something like that) then it gives more context:

    #error_handling
        shows stderr, stdout and the backtrace for unexpected errors
        OptionParser
          shows a proper error-message for [OptionParser::MissingArgument]

compared to:

    .handle_error
        shows stderr, stdout and the backtrace for unexpected errors
        for options
          shows a proper error-message for [OptionParser::MissingArgument]